docs/tutorial/images.rst

```diff
@@ -93,69 +108,58 @@ There are also several properties available for convenient access to the scan me
+    rgb = scan.get_image("rgb")  # matrix with `shape == (h, w, 3)`
+    blue = scan.get_image("blue")  # single color so `shape == (h, w)`
 
-    scan.plot(channel="red", adjustment=lk.ColorAdjustment([50, 50, 50], [100, 250, 196], gamma=[2, 1, 1]))
+    # Plot manually
+    plt.imshow(rgb)
 
-The limits can also be specified in percentiles when this is more practical::
+For scans with multiple frames::
 
-    scan.plot(channel="red", adjustment=lk.ColorAdjustment([5, 5, 5], [95, 95, 95], mode="percentile"))
+    # returned data has `shape == (n_frames, h, w, 3)`
+    rgb = multiframe_scan.get_image("rgb")
+    # returned data has `shape == (n_frames, h, w)`
+    blue = multiframe_scan.get_image("blue")
 
-The images can also be exported in the TIFF format::
+    # Manually plot the RGB image of the first frame.
+    plt.imshow(rgb[0, :, :, :])
 
-    scan.export_tiff("image.tiff")
+We can also slice out a subset of frames from an image stack::
 
-Scans can also be exported to video formats.
-Exporting the red channel of a multi-scan GIF can be done as follows for example::
+    sliced_scan = multiframe_scan[5:10]
 
-    scan.export_video("red", "test_red.gif")
+This will return a new :class:`~lumicks.pylake.scan.Scan` containing data equivalent to::
 
-Or if we want to export a subset of frames (the first frame being 10, and the last frame being 40) of all three channels
-at a frame rate of 40 frames per second, we can do this::
+    multiframe_scan.get_image("rgb")[5:10, :, :, :]
 
-    scan.export_video("rgb", "test_rgb.gif", start_frame=10, end_frame=40, fps=40)
+We can also slice the frames by time::
 
-For other video formats such as `.mp4` or `.avi`, ffmpeg must be installed. See
-:ref:`installation instructions <ffmpeg_installation>` for more information on this.
+    # get frames corresponding to the time range 30 through 90 seconds
+    sliced_scan = multiframe_scan["30s":"90s"]
 
+Or directly using timestamps::
 
-Correlating scans
------------------
+    # get frames that fall between the start and stop of a force channel
+    multiframe_scan[multiframe_file.force1x.start:multiframe_file.force1x.stop]
 
-We can downsample channel data according to the frames in a scan. We can use :func:`~lumicks.pylake.scan.Scan.frame_timestamp_ranges()` for this::
+Correlating a multiframe scan with data channels
+-------------------------------------------------
+The frames of a multiframe scan can be correlated to the force or other data channels.
+Downsample channel data according to the frames in a scan using :func:`~lumicks.pylake.scan.Scan.frame_timestamp_ranges()`::
 
-    frame_timestamp_ranges = scan.frame_timestamp_ranges()
+    frame_timestamp_ranges = multiframe_scan.frame_timestamp_ranges()
 
 This returns a list of start and stop timestamps that can be passed directly to :func:`~lumicks.pylake.channel.Slice.downsampled_to`, which will then return a :class:`~lumicks.pylake.channel.Slice` with a datapoint per frame::
```

Comment:
```suggestion
This returns a list of start and stop timestamps that can be passed directly to :func:`~lumicks.pylake.channel.Slice.downsampled_over`, which will then return a :class:`~lumicks.pylake.channel.Slice` with a datapoint per frame::
```
I assume this should be `downsampled_over`?

Reply:
Good catch, I overlooked this error in the current docs.